test/StdChains.t.sol

```diff
@@ -84,23 +84,23 @@ contract StdChainsTest is Test {
     //     _testRpc("flare_coston2");
     // }
 
-    function test_ChainNoDefault() public {
+    function test_RevertIf_ChainNoDefault() public {
```

Comment:
To answer your question from #599:

> But I think this doesn't make a lot of sense, so I was about to rename the test like this : test_RevertIf_ChainNotFound() .

Definitely feel free to rename tests so the test names read better, e.g. `test_RevertIf_ChainNotFound` makes sense to me for this one